### ISSUE TITLE
[tests] ask interaction test failing in all platforms

### ISSUE THREAD

**Issue by anfibiacreativa** [Azure-Samples/azure-search-openai-javascript#98]
> ## Please provide us with the following information:

### This issue is for a: (mark with an `x`)

```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [x] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce

run tests with 
`npx playwright test`
askinteraction times out

![Screenshot 2023-10-30 at 12 10 57 PM](https://github.com/Azure-Samples/azure-search-openai-javascript/assets/4014025/f64313b5-d4ac-4ae3-b8d7-a143140ade56)
![Screenshot 2023-10-30 at 12 11 29 PM](https://github.com/Azure-Samples/azure-search-openai-javascript/assets/4014025/6dfefd1a-3d25-457b-9749-0e1f3c28cf95)

**Comment by shibbas** [comment]
It seems to be failing because of the ask interaction itself is getting errors. 
<img width="950" alt="image" src="https://github.com/Azure-Samples/azure-search-openai-javascript/assets/26466942/db0cdb1e-8ce2-40be-9b3d-ce96d5572968">

**Comment by anfibiacreativa** [comment]
fixed with #104
